### PR TITLE
Math Magicians: tests.

### PR DESCRIPTION
**Pull request by tiagorahal**
#### In this branch:
- Add tests for operate.js and calculator.js.
- Add tests for pages.

### CODE REVIEW

File: src/logic/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  });
+
+  test('division with 0', () => {
+    expect(operate(2, 0, '÷')).toEqual('0');
+  });
+
+  test('adds two numbers', () => {
+    expect(Number(operate(2, 3, '%'))).toEqual(2);
+  });
+});
```
Comment (Emmyn5600) [inline src/logic/operate.test.js]: - [x] Here, it has to be divisible not addition, kindly correct that typo.

Comment (Emmyn5600) [inline src/logic/operate.test.js]: - [x] Great job guys, all 30 tests are passing successfully, however let us test the following in the operate.test.js file

- Should give a 0 error if given a wrong operator
- Should throw an error if given a wrong number

File: src/logic/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  });
+
+  test("when press '.' it adds the '.' if total doesn't contains '.'", () => {
+    expect(calculate({ total: '6', next: null }, '.')).toEqual({ total: '6.' });
+  });
+
+  test("when press '.' it adds the '0.' if total doesn't exists", () => {
+    expect(calculate({ total: null, next: null }, '.')).toEqual({ total: '0.' });
+  });
+});
```
Comment (Emmyn5600) [inline src/logic/calculate.test.js]: - [x]  Great job guys, all 30 tests are passing successfully.
test6

However let us test the following in the calculate.test.js file

- It should throw Zero division error.
- It should throw 0 error if trying to make a zero division
- It should give the percentage of the first operand if the buttonName is %

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @tiagorahal  and @LOctavio 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
   - The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Some tests are passing successfully.:heavy_check_mark:

Changes required :writing_hand: 
 
      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @tiagorahal @LOctavio  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**